### ISSUE TITLE
[Notification Request] Discord Release Announcements

### ISSUE THREAD

**Issue by justinthejaguar** [Kreals/TurnMarker#1]
Thanks for forking this. Is there any way to get this on the Discord Release Announcements on the Foundry VTT Discord?

**Comment by Kreals** [comment]
I'm more than happy to do that,... not exactly sure how :p, I can have a look into it.

**Comment by Kreals** [comment]
Ok, it seems i would have to message Atropos directly, not sure if that would be appreciated. There are plans to move this as the official repo for turnmarker, so might make sense to do it when that switch becomes official.

**Comment by Aeristoka** [comment]
> Ok, it seems i would have to message Atropos directly, not sure if that would be appreciated. There are plans to move this as the official repo for turnmarker, so might make sense to do it when that switch becomes official.

Would love to see this happen, I've been happy that it has been "working" even up to 0.7.9 after it was abandoned by the original dev, and would love to see you take it over officially.

**Comment by kckaiwei** [comment]
Hey @Kreals didn't realize you made a fork as well. I've adopted Turnmarker when I didn't see any updates in forever, and didn't see you had a fork as well. (Since it wasn't labelled under the original repo) I've already got ownership of the official repo on FoundryVTT and would love to merge our two together.

Mine's at:
https://github.com/kckaiwei/TurnMarker-alt

I see that you've done a rewrite, which I haven't looked into doing since my major concerns were fixing the 2 largest bugs first, and tackling any open issues from the original repo.

I see you've made a few other fixes along the way. I'd be more than happy to have you added as a maintainer so we can get all the changes merged together.

**Comment by Kreals** [comment]
Have you joined the [Discord for the League of Extraordinary FoundryVTT Developers](https://discord.gg/2rHs78h)? if not i would highly suggest it.

As for merging, sure. Any ideas on how you want to approach it?

**Comment by kckaiwei** [comment]
Yeah, I'm on there. I'll shoot a message if you want to talk more in depth as well.

I would suggest you make PRs for any new features you've made already. I would not do the refactor since your refactor actually introduced a few bugs, such as offset for the marker when using grids (since `width` was used instead of `token.w`, the two have different effects). I haven't looked through all your commits since the commit messages aren't clear, and that makes digging back a pain.

It would be best to get the new features in, and then reassess which bugfixes you did are actually needed. We're both still new to this codebase, seeing how we both created our repos recently, so I'm hesitant to do a refactor until we're a few iterations in and understand the code better.

I do have contribution guidelines as well:
https://github.com/kckaiwei/TurnMarker-alt/wiki/Contributor-Guidelines
